maxim/models/test_run.py

```
@@ -326,7 +332,7 @@ def dict_to_class(cls, data: Dict[str, Any]) -> "TestRun":
 
 @dataclass
 class TestRunEntry:
-    data_entry: Dict[str, Union[str, List[str], None]]
+    variables: Dict[str, Variable]
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)